modules/olm-bundle-format.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// * operators/understanding_olm/olm-packaging-formats.adoc
+// * operators/operator_sdk/osdk-managing-bundle-images.adoc
+
+[id="olm-bundle-format_{context}"]
+= Bundle Format
+
+The _Bundle Format_ for Operators is a new packaging format introduced by the
+Operator Framework. To improve scalability and better enable upstream users
+hosting their own catalogs, the Bundle Format specification simplifies the
+distribution of Operator metadata.
+
+An Operator bundle represents a single version of an Operator. On-disk _bundle manifests_
+are containerized and shipped as a _bundle image_, a non-runnable container
+image that stores the Kubernetes manifests and Operator metadata. Storage and
+distribution of the bundle image is then managed using existing container tools
+like `podman` and `docker` and container registries like Quay.
+
+Operator metadata can include:
+
+* Information that identifies the Operator, for example its name and version.
+* Additional information that drives the UI, for example its icon and some example Custom Resources (CRs).
+* Required and provided API(s).
```

Comment:
I don't think the parentheses are necessary here.

Reply:
:sweat_smile: